package.json

```diff
@@ -2,6 +2,18 @@
   "name": "ove",
   "private": true,
   "license": "MIT",
+   "scripts": {
+    "install": "lerna bootstrap --hoist",
+    "install:prod": "lerna bootstrap --hoist -- --production --no-optional",
+    "clean": "lerna run clean",
+    "build": "lerna run build",
+    "build:clean": "lerna run clean && lerna run build",
+    "build:docker": "lerna run clean && lerna run build && docker-compose build",
+    "test": "lerna run test",
+    "start": "OVE_HOST='localhost:8080' TUORIS_HOST='localhost:7080' pm2 start pm2.json",
+    "start:windows": "OVE_HOST='localhost:8080' TUORIS_HOST='localhost:7080' pm2 start pm2-windows.json",
```

Comment:
Not sure if we would have to use `\"` instead of `'`. We previously had to something like that for @davidbirchwork to get curl commands working on Windows.

Reply:
Maybe @davidbirchwork or somebody else with a Win machine can test this.